bin/copy_production_db_to_local

```diff
@@ -2,7 +2,7 @@
 source .env
 export RAILS_ENV=development
 bundle exec rails db:migrate db:drop db:create
-./bin/heroku_pg_dump_production_replica
+bash ./bin/heroku_pg_dump_production_replica
 pg_restore -U $PG_USERNAME --verbose --clean --no-acl --no-owner -h localhost -d code_fund_ads_development tmp/production-shallow.dump
 bundle exec rails db:migrate
-./bin/generate_db_artifacts
+bash ./bin/generate_db_artifacts
```

Comment:
Do you need to add bash here since both of these binstubs have `#!/usr/bin/env bash` at the top?

Reply:
probably not. I'll remove them